src/avt/QtVisWindow/vtk/vtkRubberBandMapper2D.C

```diff
@@ -328,6 +328,12 @@ vtkRubberBandMapper2D::RenderOverlay_X11(vtkViewport* viewport, vtkActor2D* acto
 //  Modifications:
 //    Kathleen Biagas, Mon Jun 11 15:48:10 MST 2012
 //    Remove ifdef preventing this method from bein used on _WIN32
+//
```

Comment:
Does this comment still apply since you didn't move the code from vtkRubberBandMapper2D_body.C. You can probably remove the whole comment, since there don't appear to be any changes now (other than d -> privateInstance).

Reply:
Fixed.